app/views/content/life-as-a-teacher/explore-subjects/engineers-teach-physics.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Engineers teach physics
+subject: physics
 title_paragraph: |-
   <p>
   If you're an engineer or material scientist, an engineers teach physics initial teacher training course could be a great start to your teaching journey.
```

Comment:
This intro in the yellow header is very long for ETP compared with other pages - is it worth ditching some of it or moving it to the body content instead?

Plus we don't reference the bursaries/scholarships in the header as we do on the subject pages - should we? (Happy for you to say no as this is a bit of a different page to the other subject pages - but it is a landing page)